integration-tests/src/test/java/org/zowe/apiml/ws/WebSocketProxyTest.java

```diff
@@ -126,7 +127,7 @@ public void shouldRouteHeaders() throws Exception {
     }
 
     @Test
-    @WebsocketTest @TestsNotMeantForZowe
+    @WebsocketTest
```

Comment:
Shouldn't web socket tests be excluded?

Reply:
I excluded all the WebSocket tests on the class level.